### PR TITLE
Api/upload hello world api

### PR DESCRIPTION
**Pull request by Liriaene**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by gstone271 — CHANGES_REQUESTED** [review]
Automatically-generated files should not be included in git. This includes everything in the `build` folder. (`package-lock.json` is an exception that should be included in git; see https://docs.npmjs.com/files/package-lock.json). 
Remove `build` from git and add a line to `.gitignore` so that it will not be added henceforth.

Otherwise, looks good.

**Review by gstone271 — CHANGES_REQUESTED** [review]
* We do want to include the configuration files (package, package-lock, and tsconfig)
* Let's use a wildcard to exclude everything in the build folder--I suspect future builds will generate more files here

File: src/API/app.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//
```
Comment (gstone271) [inline src/API/app.ts:1]: * Folder `src` is for the client app only; the API should not be included
* I'd rather have a lowercase folder name
So I'd just put stuff in `/api`

**Review by gstone271 — APPROVED** [review]

File: src/API/app.ts (hunk 1 above)
Comment (ElijahRW) [reply src/API/app.ts:1]: Same. this looks great otherwise.

**Review by ElijahRW — CHANGES_REQUESTED** [review]

**Review by ElijahRW — APPROVED** [review]

**Pull request merged**